index.js

```diff
@@ -175,6 +184,19 @@ alexa.response = function(session) {
   };
 };
 
+alexa.directives = function(directives) {
+  // load the alexa response directives information into details
+  this.details = directives;
+
+  this.set = function(directive) {
+    this.details.push(directive);
+  };
+
+  this.clear = function() {
```

Comment:
appreciate the tag, unfortunately, i'm more of a user/consumer on this project right now. I don't know *anything* about the internals, and haven't had the time to open up the code inside alexa-app or alexa-app-server to look at anything.  I've filed some tickets at my regular job to work on some improvements here, but at this time, they keep getting moved to the bottom of our backlog. . . and i just don't have the time to work on code outside of work right now.